md-dao/src/main/java/com/wipro/ats/bdre/md/dao/ProcessTemplateDAO.java

```diff
@@ -45,7 +45,11 @@
     private static final Logger LOGGER = Logger.getLogger(ProcessTemplateDAO.class);
     @Autowired
     SessionFactory sessionFactory;
-
+    private static final String PROCESSTEMPLATEID="processTemplateId";
+    private static final String PARENTPROCESSTEMPLATEID="processTemplate.processTemplateId";
+    private static final String DELETEFLAG="deleteFlag";
+    private static final String CONFIGGROUP="configGroup";
+    private static final String CONFIGGROUPID="id.processId";
 
     public List<com.wipro.ats.bdre.md.beans.table.ProcessTemplate> list(Integer pageNum, Integer numResults, Integer processId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "list" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)